src/main/resources/db/migration/changes/009-CourseStaff-add-Email-Orgstat-field.json

```
@@ -0,0 +1,44 @@
+{ "databaseChangeLog": [
+  {
+    "changeSet": {
+      "id": "009-CourseStaff-add-email-orgstat-fields",
+      "author": "JonahS",
```

Let's make the id match the filename, and the author match the githubid.

```suggestion
      "id": "009-CourseStaff-add-Email-orgstat-field",
      "author": "jonahso",
```